Analysis/source/bbetauAnalyzer.cxx

```diff
@@ -13,9 +13,9 @@ class bbetauAnalyzer : public BaseEventAnalyzer<ElectronCandidate, TauCandidate>
 protected:
     virtual EventRegion DetermineEventRegion(EventInfo& event, EventCategory /*eventCategory*/) override
     {
-        static const std::vector<std::string> trigger_patterns = {
-            "HLT_Ele25_eta2p1_WPTight_Gsf_v"
-        };
+
+
+        static const std::vector<std::string> trigger_patterns = ana_setup.trigger.at(ChannelId());
```

Comment:
now check for trigger patterns are common for all channels, right? If yes, it should be moved to the BaseAnalyzer.